src/styles/product-card.scss

```diff
@@ -0,0 +1,66 @@
+.product-card {
+  width: $height-product-card;
+  height: 100%;
+
+  background-color: $color-white;
+  border: 1px solid $main-border;
+  border-radius: 5px;
+  transition: $product-card-transition;
+
+  &:hover {
+    transform: $product-card-transform;
+  }
+
+  &__name {
+    margin: 35px auto 4px;
+  }
+
+  &:hover &__name {
+    color: $hover-title-card-color;
+  }
+
+  &__container {
+    padding: 32px 16px 16px;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: auto 3px;
+  }
+
+  &__code {
+    @include secondary-styles;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+
+    @include secondary-styles;
+
+    &::after {
+      content: "Reviews: 5";
```

Comment:
Consider just placing the review block inside your HTML.

Reply:
it's the right thing to do, right?